### PR TITLE
Handle Asgard as street_network backend

### PR DESCRIPTION
**Pull request by mbounabi**
- Add new instance for ArtemisNG with Asgard
- Add a project name in .env for ArtemisNG
- Modify Jinja template to handle custom Jormun settings

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,6 @@
+COMPOSE_PROJECT_NAME=navitia-docker-compose
+
 TAG=latest
 KIRIN_TAG=latest
-
 ASGARD_DATA_TAG=france
 ASGARD_APP_TAG=latest
```
Comment (AurelienLP) [inline .env]: Is it possible to put master here ? Tag latest is never updated while master is after every merge.

Comment (mbounabi) [reply .env]: For ASGARD_APP_TAG? Sure

Comment (AurelienLP) [reply .env]: Thanks!

File: .env [hunk 2]
```diff
@@ -1,5 +1,6 @@
+COMPOSE_PROJECT_NAME=navitia-docker-compose
```
Comment (kinnou02) [inline .env]: This might be a bit long to type

Comment (mbounabi) [reply .env]: This is actually the default name. I just wanted to make sure it never changes.
Will just 'navitia' be enough?

**Pull request merged**